disperser/common/v2/blobstore/dynamo_metadata_store.go

```diff
@@ -50,6 +50,11 @@ var (
 	ErrInvalidStateTransition = errors.New("invalid state transition")
 )
 
+type StatusIndexCursor struct {
+	PK        string
+	UpdatedAt uint64
+}
```

Comment:
Low priority ask, could you add a short godoc for `PK`, it's not obvious at first glance what it means.

Reply:
Actually, you made me realize `PK` is probably not the right field to export from the cursor. Will change it to `BlobKey`